src/Compilers/Core/MSBuildTask/project.json

```diff
@@ -12,6 +12,7 @@
         "System.IO.FileSystem": "4.0.1",
         "System.IO.FileSystem.DriveInfo": "4.0.0",
         "System.IO.Pipes": "4.0.0",
+        "System.Reflection": "4.1.0",
```

Comment:
@jaredpar Yup. In the compiler extension vsix project you'll need to

1. Add an entry for NuGetPackagesToInclude that includes `System.Reflection`.
2. If this has a facade, you'll need to also manually include the facade.
3. You'll also need to add a codebase redirect in that project to point to the facade.

If there is a facade you'll also need to add the facade assembly to the list of files in ` Src/Setup/DevDivVsix/CompilersPackage/Microsoft.CodeAnalysis.Compilers.swr`